rbd/rbd.go

```diff
@@ -194,7 +194,7 @@ func GetError(err C.int) error {
 	}
 }
 
-//
+// Version returns the major, minor, and patch level of the librbd library.
```

Comment:
Missing the "Implements:" part :-)

Reply:
I don't think that's needed for this one.